package.json

```diff
@@ -17,6 +17,7 @@
     "build:compile": "babel --source-maps -d lib src",
     "build:bundle": "NODE_ENV=production webpack -p lib/vector/index.js vector/bundle.js",
     "build": "npm run build:css && npm run build:compile && npm run build:bundle",
+    "package": "npm run build && mkdir packages && bestzip packages/vector-`git describe --dirty || echo unknown`.zip vector",
```

Comment:
mkdir is on Windows, mkdir -p is not - hence plain mkdir, although I realise of course && won't work so I've replaced it with ';'. We don't rely on git - we just use 'unknown' if the git command doesn't work (nonetheless it is very tempting to drop windows support for this given it's so annoying, but this just about seems doable).